geoip2/models.py

```diff
@@ -577,6 +577,23 @@ class ISP(ASN):
 
       :type: unicode
 
+    .. attribute: mobile_country_code
+
+      The `mobile country code (MCC)
+      <https://en.wikipedia.org/wiki/Mobile_country_code>`_ associated with the
+      IP address and ISP.
+
+      :type: str
+
+    .. attribute: mobile_network_code
+
+      The `mobile network code (MCC)
+      <https://en.wikipedia.org/wiki/Mobile_country_code>`_ associated with the
```

Comment:
Is the URL correct?

Reply:
Yes, I think it is. Miguel provided it and I didn't see a better one.